tasks/install-matlab/v1-beta/src/matlab.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 The MathWorks, Inc.
+
+import * as toolLib from "azure-pipelines-tool-lib/tool";
+import * as fs from "fs";
+import * as https from "https";
+import * as script from "./script";
+
+export interface IRelease {

```

Comment:
eslint is built into the pipeline for this action so it was failing because apparently this is a typescript best practice. I can update the config to have it ignore this rule. I am not really a fan of the "I" prefix either.